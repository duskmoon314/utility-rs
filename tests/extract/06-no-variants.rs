use utility_types::Extract;

#[derive(Debug, PartialEq, Extract)]
#[extract(arg(
    ident = Terrestrial,
    derive(Debug, PartialEq, Clone, Copy)
))]
pub enum Planet {
    Mercury,
    Venus,
    Earth,
    Mars,
    Jupiter,
    Saturn,
    Uranus,
    Neptune,
}

fn main() {}
