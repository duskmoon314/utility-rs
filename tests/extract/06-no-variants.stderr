error: Missing field `variants`
 --> tests/extract/06-no-variants.rs:4:11
  |
4 |   #[extract(arg(
  |  ___________^
5 | |     ident = Terrestrial,
6 | |     derive(Debug, PartialEq, Clone, Copy)
7 | | ))]
  | |_^
